### PR TITLE
11.0 mig project_timeline

### PR DESCRIPTION
**Pull request by Maartincm**
Migration of the Module **project_timeline** from V10 to V11.

This module has a dependency on **web_timeline**, so this should follow the *PR* https://github.com/OCA/web/pull/912

### CODE REVIEW

**Comment by rafaelbn** [comment]
WIP until https://github.com/OCA/web/pull/912 is merged! Thanks!

**Review by pedrobaeza — APPROVED** [review]
Working flawlessly, thanks!

**Comment by pedrobaeza** [comment]
@rafaelbn I have added the temporary branch so you can test this PR.

File: project_timeline/README.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Usage
+=====
+
+* Go to *Project > Search > Tasks*.
+* Click on the icon with a clock.
+* You will see the tasks in the new view.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/10.0
```
Comment (chienandalu) [inline project_timeline/README.rst]: s/10.0/11.0

**Review by chienandalu — APPROVED** [review]
Looks good. Just a little change

**Comment by pedrobaeza** [comment]
@dreispt this can't be merged yet. The dependency is not merged and it contains an invalid commit. I'm going to rewrite a bit the commit history in the main branch for amending this.

**Comment by pedrobaeza** [comment]
Amended and merged the dependency.

**Comment by pedrobaeza** [comment]
@dreispt, please check next time the label "Work in progress" before merging.

**Comment by dreispt** [comment]
@pedrobaeza OK, sorry.

**Pull request merged**